runner/host/lsmod_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) HashiCorp, Inc.
+package host
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/hcdiag/op"
+
+	"github.com/hashicorp/hcdiag/runner"
+	"github.com/stretchr/testify/assert"
+)
+
+type sampleShellRunner struct {
```

Comment:
I wouldn't hold up merging this PR, but I had an idea. It looks like this is identical to the `mockShellRunner` in fstab_test.go. Technically, because these files are in the same package, the lsmod_test.go file could use that same mockShellRunner without having to duplicate it and give it a new name in this file. However, since it is shared, I wonder if we should move the mockShellRunner out of fstab_test.go and into host_test.go, and then it would just be a shared struct type that any test in the package could use.

Reply:
Yes that's exactly what it is. I can try to implement that and make sure it works.

Reply:
I made a new commit with the recommended changes if you wanna take a look.